test/MUXControlsTestApp/MUXControlsTestApp.Shared.shproj

```diff
@@ -4,11 +4,10 @@
   <PropertyGroup Label="Globals">
     <ProjectGuid>{6AA772A6-CBF7-4FF3-8864-BC9366015DC2}</ProjectGuid>
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
-  </PropertyGroup>
+  </PropertyGroup>  
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.Default.props" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.props" />
-  <PropertyGroup />
```

Comment:
Wow. I am surprised we were even able to build with this error!

Reply:
That is just considered as an empty PropertyGroup tag.

---
In reply to: [301743667](https://github.com/microsoft/microsoft-ui-xaml/pull/988#discussion_r301743667) [](ancestors = 301743667)